loadgen/workflow_executor.go

```diff
@@ -0,0 +1,41 @@
+package loadgen
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	"go.temporal.io/sdk/client"
+)
+
+// WorkflowExecutor is a very dumb executor that does nothing but run the specified workflow.
+// Scenarios using it are expected to implement all their driver logic as a workflow.
+type WorkflowExecutor struct {
+	WorkflowType       string
+	WorkflowArgCreator func(info ScenarioInfo) []interface{}
+}
+
+func (w WorkflowExecutor) Run(ctx context.Context, info ScenarioInfo) error {
+	// TODO: ??
+	//executorTimeout := info.ScenarioOptionDuration(scenarios.ExecutorTimeoutFlag, 1*time.Hour)
+	executorTimeout := 1 * time.Hour
+	workflowOpts := client.StartWorkflowOptions{
+		ID:                       fmt.Sprintf("%s-driver", info.TaskQueue()),
+		TaskQueue:                info.TaskQueue(), // TODO: Should be special tq for driver?
```

Comment:
Cleanup is done by task queue, so I like the idea that everything shares a task queue where we can, though there will surely be cross-task-queue scenarios in the future

Reply:
Kept it the same